features/page_objects/litigator_transfer_claim_form_page.rb

```diff
@@ -1,10 +1,13 @@
 require_relative 'sections/lgfs_transfer_detail_section'
 require_relative 'sections/lgfs_transfer_fee_section'
 require_relative 'litigator_claim_form_page'
+require_relative 'sections/common_autocomplete_section'
 
 class LitigatorTransferClaimFormPage < LitigatorClaimFormPage
   set_url "/litigators/transfer_claims/new"
 
   section :transfer_detail, LgfsTransferDetailSection, "#transfer-detail"
   section :transfer_fee, LgfsTransferFeeSection, "#transfer-fee"
+  section :auto_one, CommonAutocomplete, "#cc-transfer-stage"
+  section :auto_two, CommonAutocomplete, "#cc-case-conclusion"
```

Comment:
For clarity could these just be...
```ruby
section :transfer_stage, CommonAutocomplete, "#cc-transfer-stage"
section :case_conclusion, CommonAutocomplete, "#cc-case-conclusion"
```
?
It should make future debugging easier

Reply:
arg.. I changed the IDs and never the name.. will do